### ISSUE TITLE
[Crash] Crash on startup with BYG

### ISSUE THREAD

**Issue by kawaiicakes** [SuperMartijn642/Fusion#1]
**Version Info**
- Minecraft 1.19.2
- Rechiseled 1.1.0
- Fusion 1.0.0
- Forge 43.2.11
- BYG 2.0.1.3

**Steps to reproduce**
1. Boot the game with BYG 2.0.1.3, Rechiseled 1.1.0 and Fusion 1.0.0 installed.

**Crash report**
```
java.lang.NullPointerException: Registry Object not present: byg:quartz_crystal
	at java.util.Objects.requireNonNull(Objects.java:334) ~[?:?] {re:mixin}
	at net.minecraftforge.registries.RegistryObject.get(RegistryObject.java:204) ~[forge-1.19.2-43.2.11-universal.jar%23574!/:?] {re:mixin,re:classloading}
	at potionstudios.byg.reg.forge.ForgeRegistrationFactory$Provider$1.get(ForgeRegistrationFactory.java:121) ~[Oh_The_Biomes_You'll_Go-forge-1.19.2-2.0.1.3.jar%23511!/:2.0.1.3] {re:classloading}
	at potionstudios.byg.reg.BlockRegistryObject$1.get(BlockRegistryObject.java:87) ~[Oh_The_Biomes_You'll_Go-forge-1.19.2-2.0.1.3.jar%23511!/:2.0.1.3] {re:classloading}
	at potionstudios.byg.reg.BlockRegistryObject$1.get(BlockRegistryObject.java:74) ~[Oh_The_Biomes_You'll_Go-forge-1.19.2-2.0.1.3.jar%23511!/:2.0.1.3] {re:classloading}
	at net.minecraft.world.level.pathfinder.WalkNodeEvaluator.handler$bkh000$byg_avoidQuartzCrystals(WalkNodeEvaluator.java:1134) ~[client-1.19.2-20220805.130853-srg.jar%23569!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:canary.mixins.json:ai.pathing.WalkNodeEvaluatorMixin,pl:mixin:APP:byg.mixins.json:common.MixinWalkNodeProcessor,pl:mixin:APP:dramaticdoors.mixins.json:LandPathNodeMakerMixin,pl:mixin:APP:diagonalfences.common.mixins.json:WalkNodeEvaluatorMixin,pl:mixin:A}
	at net.minecraft.world.level.pathfinder.WalkNodeEvaluator.m_77643_(WalkNodeEvaluator.java:509) ~[client-1.19.2-20220805.130853-srg.jar%23569!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:canary.mixins.json:ai.pathing.WalkNodeEvaluatorMixin,pl:mixin:APP:byg.mixins.json:common.MixinWalkNodeProcessor,pl:mixin:APP:dramaticdoors.mixins.json:LandPathNodeMakerMixin,pl:mixin:APP:diagonalfences.common.mixins.json:WalkNodeEvaluatorMixin,pl:mixin:A}
	at net.minecraft.world.level.block.state.BlockBehaviour$BlockStateBase.handler$bfb002$init(BlockBehaviour.java:2909) ~[client-1.19.2-20220805.130853-srg.jar%23569!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:mixins.oculus.json:MixinBlockStateBehavior,pl:mixin:APP:kubejs-common.mixins.json:BlockStateBaseMixin,pl:mixin:APP:notreepunching.common.mixins.json:AbstractBlockStateAccessor,pl:mixin:APP:starlight.mixins.json:common.blockstate.BlockStateBaseMixin,pl:mixin:APP:canary.mixins.json:ai.pathing.BlockStateBaseMixin,pl:mixin:APP:canary.mixins.json:block.flatten_states.BlockStateBaseMixin,pl:mixin:APP:canary.mixins.json:util.block_tracking.BlockStateBaseMixin,pl:mixin:APP:byg.mixins.json:common.block.MixinBlockBehaviourBlockStateBase,pl:mixin:APP:blockskills-common.mixins.json:MixinBlockStateBase,pl:mixin:APP:blockskills-common.mixins.json:MixinBlockStateBaseHarvestable,pl:mixin:APP:ferritecore.blockstatecache.mixin.json:BlockStateBaseMixin,pl:mixin:A}
	at net.minecraft.world.level.block.state.BlockBehaviour$BlockStateBase.m_60611_(BlockBehaviour.java:471) ~[client-1.19.2-20220805.130853-srg.jar%23569!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:mixins.oculus.json:MixinBlockStateBehavior,pl:mixin:APP:kubejs-common.mixins.json:BlockStateBaseMixin,pl:mixin:APP:notreepunching.common.mixins.json:AbstractBlockStateAccessor,pl:mixin:APP:starlight.mixins.json:common.blockstate.BlockStateBaseMixin,pl:mixin:APP:canary.mixins.json:ai.pathing.BlockStateBaseMixin,pl:mixin:APP:canary.mixins.json:block.flatten_states.BlockStateBaseMixin,pl:mixin:APP:canary.mixins.json:util.block_tracking.BlockStateBaseMixin,pl:mixin:APP:byg.mixins.json:common.block.MixinBlockBehaviourBlockStateBase,pl:mixin:APP:blockskills-common.mixins.json:MixinBlockStateBase,pl:mixin:APP:blockskills-common.mixins.json:MixinBlockStateBaseHarvestable,pl:mixin:APP:ferritecore.blockstatecache.mixin.json:BlockStateBaseMixin,pl:mixin:A}
	at net.minecraftforge.registries.GameData$BlockCallbacks.onBake(GameData.java:467) ~[forge-1.19.2-43.2.11-universal.jar%23574!/:?] {re:classloading}
	at net.minecraftforge.registries.ForgeRegistry.bake(ForgeRegistry.java:635) ~[forge-1.19.2-43.2.11-universal.jar%23574!/:?] {re:classloading}
	at net.minecraftforge.registries.GameData.lambda$revertTo$11(GameData.java:296) ~[forge-1.19.2-43.2.11-universal.jar%23574!/:?] {re:mixin,re:classloading,pl:mixin:APP:supermartijn642corelib.mixins.json:GameDataMixin,pl:mixin:A}
	at com.google.common.collect.HashBiMap.forEach(HashBiMap.java:572) ~[guava-31.0.1-jre.jar%23121!/:?] {}
	at net.minecraftforge.registries.GameData.revertTo(GameData.java:296) ~[forge-1.19.2-43.2.11-universal.jar%23574!/:?] {re:mixin,re:classloading,pl:mixin:APP:supermartijn642corelib.mixins.json:GameDataMixin,pl:mixin:A}
	at net.minecraftforge.registries.GameData.postRegisterEvents(GameData.java:358) ~[forge-1.19.2-43.2.11-universal.jar%23574!/:?] {re:mixin,re:classloading,pl:mixin:APP:supermartijn642corelib.mixins.json:GameDataMixin,pl:mixin:A}
	at net.minecraftforge.common.ForgeStatesProvider.lambda$new$4(ForgeStatesProvider.java:25) ~[forge-1.19.2-43.2.11-universal.jar%23574!/:?] {re:classloading}
	at net.minecraftforge.fml.ModLoader.lambda$dispatchAndHandleError$23(ModLoader.java:207) ~[fmlcore-1.19.2-43.2.11.jar%23570!/:?] {}
	at java.util.Optional.ifPresent(Optional.java:178) ~[?:?] {re:mixin}
	at net.minecraftforge.fml.ModLoader.dispatchAndHandleError(ModLoader.java:207) ~[fmlcore-1.19.2-43.2.11.jar%23570!/:?] {}
	at net.minecraftforge.fml.ModLoader.lambda$gatherAndInitializeMods$14(ModLoader.java:185) ~[fmlcore-1.19.2-43.2.11.jar%23570!/:?] {}
	at java.lang.Iterable.forEach(Iterable.java:75) ~[?:?] {re:mixin}
	at net.minecraftforge.fml.ModLoader.gatherAndInitializeMods(ModLoader.java:185) ~[fmlcore-1.19.2-43.2.11.jar%23570!/:?] {}
	at net.minecraftforge.client.loading.ClientModLoader.lambda$begin$1(ClientModLoader.java:91) ~[forge-1.19.2-43.2.11-universal.jar%23574!/:?] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraftforge.client.loading.ClientModLoader.lambda$createRunnableWithCatch$4(ClientModLoader.java:111) ~[forge-1.19.2-43.2.11-universal.jar%23574!/:?] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraftforge.client.loading.ClientModLoader.begin(ClientModLoader.java:91) ~[forge-1.19.2-43.2.11-universal.jar%23574!/:?] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.<init>(Minecraft.java:468) ~[client-1.19.2-20220805.130853-srg.jar%23569!/:?] {re:mixin,pl:accesstransformer:B,xf:fml:xaeroworldmap:xaero_wm_minecraft_runtick,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,xf:fml:xaeroworldmap:xaero_wm_minecraft_runtick,pl:mixin:APP:kubejs-common.mixins.json:MinecraftMixin,pl:mixin:APP:rubidium.mixins.json:core.MixinMinecraftClient,pl:mixin:APP:balm.mixins.json:MinecraftMixin,pl:mixin:APP:chat_heads.mixins.json:MinecraftMixin,pl:mixin:APP:mixin.cleancut.json:MinecraftMixin,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_Keybinds,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_Images,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_PipelineManagement,pl:mixin:APP:playerrevive.mixins.json:MinecraftAccessor,pl:mixin:APP:cgm.mixins.json:client.MinecraftMixin,pl:mixin:APP:rubidium_extras.mixins.json:FrameCounter.FpsAccessorMixin,pl:mixin:APP:flywheel.mixins.json:PausedPartialTickAccessor,pl:mixin:APP:unionlib.mixins.json:client.MinecraftMixin,pl:mixin:APP:bookshelf.common.mixins.json:client.AccessorMinecraft,pl:mixin:APP:architectury.mixins.json:MixinMinecraft,pl:mixin:APP:quark.mixins.json:client.MinecraftMixin,pl:mixin:APP:create.mixins.json:client.WindowResizeMixin,pl:mixin:APP:securitycraft.mixins.json:camera.MinecraftMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.main.Main.m_239872_(Main.java:176) ~[client-1.19.2-20220805.130853-srg.jar%23569!/:?] {re:classloading,re:mixin,pl:runtimedistcleaner:A,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.main.Main.main(Main.java:51) ~[client-1.19.2-20220805.130853-srg.jar%23569!/:?] {re:classloading,re:mixin,pl:runtimedistcleaner:A,pl:mixin:A,pl:runtimedistcleaner:A}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?] {}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?] {}
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?] {}
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?] {}
	at net.minecraftforge.fml.loading.targets.CommonClientLaunchHandler.lambda$launchService$0(CommonClientLaunchHandler.java:27) ~[fmlloader-1.19.2-43.2.11.jar%23101!/:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:30) [modlauncher-10.0.8.jar%2388!/:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53) [modlauncher-10.0.8.jar%2388!/:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71) [modlauncher-10.0.8.jar%2388!/:?] {}
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:106) [modlauncher-10.0.8.jar%2388!/:?] {}
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:77) [modlauncher-10.0.8.jar%2388!/:?] {}
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26) [modlauncher-10.0.8.jar%2388!/:?] {}
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23) [modlauncher-10.0.8.jar%2388!/:?] {}
	at cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141) [bootstraplauncher-1.1.2.jar:?] {}```

**Comment by SuperMartijn642** [comment]
The game seems to start fine with the mods and versions you mentioned.
Looking at the log you added, nothing in there seems to be related to anything Fusion does.

**Comment by kawaiicakes** [comment]
> The game seems to start fine with the mods and versions you mentioned. Looking at the log you added, nothing in there seems to be related to anything Fusion does.

Interesting. Something is definitely causing the game to crash when I have Fusion installed. I tried launching twice with Fusion installed, then deleted it and reverted versions on Rechiseled and was finally able to launch. 

Would you like me to send the full crash log? I avoid doing so where possible because I'm afraid some personal identifiers may be in there.

**Comment by SuperMartijn642** [comment]
The full log would be nice, preferably the `latest.log` file. The only thing that would be in there is your sessionId, but Forge replaces it with stars (*).

**Comment by DelviousCrafts** [comment]
This could be caused by an issue I also ran into. The crash seems to be caused by create, but it seems to cause a weird cascade crash with all kinds of mods that do any kind of terrain manipulation. 

[initial crash](https://gist.github.com/DelviousCrafts/d3daffee4b1812552ed0aceb0ca963d9) on my main modpack that indicated a problem. 

[crash](https://gist.github.com/DelviousCrafts/8ad8e6457300497eff9b49bc37289ab3) with nothing but re-chiseled, create, JEI, and required mods.

Ran every other mod in the initial crash stating it was having an error and had absolutely no issues running them alone with the latest re-chiseled update. 

In case it helps
[latest log](https://gist.github.com/DelviousCrafts/9fdbdba33e8bd5686247e1160c3f0a3d) with full modpack
[latest log](https://gist.github.com/DelviousCrafts/6c237ccb7e2e91e647ddd99dd2b0ad7b) with only create, re-chiseled, JEI, and required mods.

Create version: 0.5.0.1.b

**Comment by kawaiicakes** [comment]
> The full log would be nice, preferably the `latest.log` file. The only thing that would be in there is your sessionId, but Forge replaces it with stars (*).

Unfortunately I don't think I have the latest.log from the crash. Only the crash report. I can try forcing the crash again some time if the cause of the crash isn't found in the crash report.

https://pastebin.com/9YeVQeTz 

I have this set to delete in one week. Please save it if you need it for longer than that.

**Comment by SuperMartijn642** [comment]
@DelviousCrafts that issue is fixed now in version 1.1.0a of Rechiseled. It is however unrelated to the crash from @kawaiicakes.

@kawaiicakes the crash log doesn't seem to show anything either. You are using too many mods for me to test it. Could you narrow down the mods needed to reproduce the issue?

**Comment by kawaiicakes** [comment]
> @kawaiicakes the crash log doesn't seem to show anything either. You are using too many mods for me to test it. Could you narrow down the mods needed to reproduce the issue?

I've just tried the latest update you released and I was finally able to launch. Perhaps there was some strange interaction between Create, BYG, and Re-Chiseled (or potentially other mods) as I _did_ have Create installed.

Thank you for your time. I am now closing this issue. @SuperMartijn642
